### ISSUE TITLE
/exit kicks character all the way out

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#170]
![image](https://user-images.githubusercontent.com/52435234/64841951-a46b8580-d5ce-11e9-8c10-548a799e93e1.png)

exit seems to be a wee bit broke again
- `/exit` did not work & made the entire room glitch out
- `/exit` kicked my character aaaaallll the way out

![image](https://user-images.githubusercontent.com/52435234/64842027-da106e80-d5ce-11e9-9264-f3ea79a35265.png)

maybe a minor issue, possibly related to #152 & #158

_Originally posted by @lostamongtrees in https://github.com/RolePlayGateway/roleplaygateway/issues/96#issuecomment-531117019_

**Comment by lostamongtrees** [comment]
- `/exit`
### no
![image](https://user-images.githubusercontent.com/52435234/64844493-f368e900-d5d5-11e9-8bcc-4fdd356a584c.png)

- `/exit`
![image](https://user-images.githubusercontent.com/52435234/64844589-3925b180-d5d6-11e9-92d7-1d109ffff273.png)
- `/look`
![image](https://user-images.githubusercontent.com/52435234/64844616-4a6ebe00-d5d6-11e9-9601-8b3ff662b981.png)

and the `Shadowmine` the `vehicle` is still here a la the additions to  #152 which lead up to this.

**Comment by lostamongtrees** [comment]
seems to be working! cannot replicate issue, and `/exit` places character where it should
